lib/test-worker.js

```diff
@@ -86,6 +86,12 @@ var oldNodeModulesPaths = module.constructor._nodeModulePaths;
 module.constructor._nodeModulePaths = function () {
 	var ret = oldNodeModulesPaths.apply(this, arguments);
 	ret.push(nodeModulesDir);
+	if (process.env.NODE_PATH) {
+		var osSplitChar = /^win/.test(process.platform) ? ';' : ',';
```

Comment:
No need to use `is-windows`, since all of its logic is 1 line (https://github.com/jonschlinkert/is-windows/blob/master/index.js#L20):

```js
process.platform === 'win32'
```